### PR TITLE
Make a more flexible customize process

### PR DESCRIPTION
**Pull request by DrDaveD**
This was inspired by a need to add an additional customization in the tiger kubernetes cluster.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,10 +12,12 @@ RUN groupadd -o -g 10941 squid && \
     useradd -o -u 10941 -g 10941 -s /sbin/nologin -d /var/lib/squid squid && \
     yum update -y && \
     yum install -y frontier-squid --enablerepo=osg-testing && \
-    rm -rf /var/cache/yum/*
+    rm -rf /var/cache/yum/* && \
+    mkdir /etc/squid/customize.d
```
Comment (brianhlin) [inline Dockerfile:16]:
```suggestion
    rm -rf /var/cache/yum/*
    
RUN mkdir /etc/squid/customize.d
```

It's nice to have this in another layer

**Review by brianhlin — COMMENTED** [review]
Just some minor comments. I also gave you GitHub `triage` permissions so you can request reviewers now

File: Dockerfile (hunk 1 above)
Comment (DrDaveD) [reply Dockerfile:16]: I thought layers were bad because they cause an extra layer of overlayfs.  What is the benefit of having a layer just for that?

Comment (brianhlin) [reply Dockerfile:16]: Ah I didn't think about that. I usually prefer the extra layers during development to make use of the cache but it probably makes sense to join things together in the finished product!

**Comment by brianhlin** [comment]
FYI @efajardo @Mansalu

**Review by brianhlin — APPROVED** [review]

**Comment by Mansalu** [comment]
IIRC we hacked something similar into the SLATE chart a while back.

**Comment by DrDaveD** [comment]
What's this waiting for?  I thought it was already in.

**Comment by brianhlin** [comment]
@DrDaveD that's my bad. Merged.

**Comment by DrDaveD** [comment]
Ok, cool.  After it is built into the current docker container, the [documentation](https://opensciencegrid.org/docs/data/run-frontier-squid-container/) should also refer to it.

**Comment by brianhlin** [comment]
Hi @DrDaveD , the `fresh` image has been built automatically on the merge to master: https://hub.docker.com/r/opensciencegrid/frontier-squid/tags?page=1&ordering=last_updated&name=fresh. If you submit the PR to update the docs we can merge it once someone verifies its functionality and we promote it to `stable`

**Pull request merged**